evennia/commands/default/building.py

```diff
@@ -2075,6 +2100,9 @@ def func(self):
                 new_typeclass, clean_attributes=reset, clean_cmdsets=reset, run_start_hooks=hooks
             )
 
+            if "prototype" in self.switches:
+                spawner.batch_update_objects_with_prototype(prototype, objects=[obj])
```

Comment:
The `batch_update_objects_with_prototype` returns the number of objects modified by this operation. In this case it will either be 1 or 0, if there was an error. This feels like a good thing to catch so errors can be reported back to the user. 
It may also be good to modify the success message as well, to inform the user that the prototype this-and-that was successfully applied.

Reply:
Good suggestions. Added.